dashboard/app/views/home/_privacy.md

```diff
@@ -39,7 +39,8 @@ Data stored by Code.org if you create a Code.org student account | Required? | H
 --- | --- | ---
 Display Name (eg “Cool Coder” or “John”) | Mandatory | To provide students a welcome upon login and to identify the student in the teacher’s view of student progress. We recommend using first names only to increase privacy.
 Age (*Not* birthdate) | Mandatory | To understand the developmental stage of students to offer an age-appropriate experience for each student. We use this field to ensure we don’t allow students under age 13 to access age-inappropriate features (such as sharing their coding projects on social media). We collect ages (e.g. 16), as opposed to birth dates (e.g. Feb 13, 1998).
-1-way hash of email address (NOT the actual email addresses, which are collected in the web browser but never transmitted to Code.org and thus never stored by us) | Optional if account created by teacher or parent; Mandatory if student signs up directly| If a student creates an account directly, an email address is required. If the account is created by a teacher or parent, it is optional.<br><br>Even if provided, student email addresses are NOT stored by Code.org in a retrievable format. To protect student privacy, we only store a 1-way hash of the email address. We do not have any way of sending email to these students or retrieving their actual email address. It is used only for purposes of login. See section on Student Email Addresses below for more details.
+1-way hash of student email address (NOT the actual email addresses, which are collected in the web browser but never transmitted to Code.org and thus never stored by us) | Optional if account created by teacher or parent; Mandatory if student signs up directly| If a student creates an account directly, an email address is required. If the account is created by a teacher or parent, it is optional.<br><br>Even if provided, student email addresses are NOT stored by Code.org in a retrievable format. To protect student privacy, we only store a 1-way hash of the email address. We do not have any way of sending email to these students or retrieving their actual email address. It is used only for purposes of login. See section on Student Email Addresses below for more details.
+Parent email address | Optional - parent can use their email to  create a login for their child at home | Students may want to create a personal login to keep using their teacher-created account after the school year ends.<br><br>If the student is under 13 or doesn’t have a personal email address, we require a parent email address for password recovery. The student will then be able to sign in at home with a username and password. 
```

Comment:
You're going to hate me. There's a double space (in both PRs).

`email to__create`